contact.html

```diff
@@ -0,0 +1,622 @@
+
+                                                  </span>
+                                                </a>
+                                              </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="container responsivegrid contact-form aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div id="contact-container" class="cmp-container">
+                          <div class="text formtext aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="text-cdbe0aaaa2" class="cmp-text">
+                                <p>This is a safe place to learn, grow, and ask quesitons. We will protect your identity and never use this information for marketing purposes.</p>
+                            </div>
+                          </div>
+                          <div class="container responsivegrid form-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="contact-container" class="cmp-container">
+                              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+                                <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                  <form id="new_form" name="new_form" enctype="multipart/form-data" class="cmp-form aem-Grid aem-Grid--12 aem-Grid--default--12">
+                                    <div class="text full-name aem-GridColumn aem-GridColumn--default--12">
+                                      <div class="cmp-form-text">
+                                        <input class="text-mail" type="text" id="fname" name="firstname" placeholder="Name">
+                                      </div>
+                                    </div>
+                                    <div class="text mail-field aem-GridColumn aem-GridColumn--default--6">
+                                      <div class="cmp-form-text" data-cmp-required-message="You need to add your email to log in">
+                                        <input class="mail" type="email" id="email" name="email" placeholder="Email">
+                                      </div>
+                                    </div>
+                                    <div class="text phone-field aem-GridColumn aem-GridColumn--default--6">
+                                      <div class="cmp-form-text">
+                                        <input class="phone" type="tel" id="phone" name="phone" pattern="[0-9]{3}-[0-9]{2}-[0-9]{3}" placeholder="Phone">
```

Comment:
@Prystal254 Needs a `.cmp-form-text__text` class and `.phone` can't go here. Custom classes must be on the root of the component.